packages/react/tsconfig.json

```diff
@@ -1,14 +1,16 @@
 {
   "compilerOptions": {
+    "moduleResolution": "node",
+    "baseUrl": ".",
     "declaration": true,
     "esModuleInterop": true,
     "jsx": "react-jsx",
     "rootDir": "./src",
     "outDir": "./dist",
     "skipLibCheck": true,
     "sourceMap": true,
-    "module": "commonjs",
-    "target": "es5"
+    "module": "esnext",
```

Comment:
So, TIL : https://stackoverflow.com/questions/41993811/understanding-target-and-module-in-tsconfig

`module` basically means `module.exports` or `import/export` or `import()`:
> https://github.com/microsoft/TypeScript/issues/24082